src/cmd/hotspots/assert.rs

```diff
@@ -38,32 +52,54 @@ impl Cmd {
         let client = helium_api::Client::new_with_base_url(api_url(wallet.public_key.network));
 
         let wallet_key = keypair.public_key();
+        // Get the next likely gateway nonce for the new transaction
+        let nonce = helium_api::hotspots::get(&client, &self.gateway.to_string())
+            .await?
+            .nonce
+            + 1;
+        let payer = if self.onboarding {
+            staking_client.address_for(&self.gateway).await?.into()
+        } else {
+            vec![]
```

Comment:
In #hip-12-remote-assert on Discord. @hashc0de helped me by providing this diff:
```
-            vec![]
+            wallet_key.into()
```
Making this change, allowed me to actually submit an assert_v2 transaction that was accepted.